tests/ipc_stress.c

```diff
@@ -110,9 +109,12 @@ int main()
 
     for (i = 0; i != THREAD_COUNT; ++i)
         nn_thread_init(&cli_threads[i], client, (void *)(intptr_t)i);
+
     for (i = 0; i != THREAD_COUNT; ++i)
         nn_thread_term(&cli_threads[i]);
 
+    nn_thread_term(&srv_thread);
```

Comment:
This change looks unrelated.

Reply:
@gdamore suggesting another pull request? 
For me, it is related, because I've checked changes with thread sanitizer. 
We have thread leak in this case.

Reply:
Right.  That's another issue.  It could be a different commit, I suppose, which would be slightly better... I'm willing to be a bit lax on that though.